lib/confluence/failure_to_ship_data.es6.js

```diff
@@ -11,48 +11,45 @@ foam.CLASS({
   ids: ['browserName', 'date'],
   properties: [
     {
-      name: 'browserName',
       class: 'String',
-      documentation: 'Name of the Browser',
+      name: 'browserName',
+      documentation: 'Name of the Browser.',
       required: true,
       final: true,
     },
     {
+      class: 'FObjectProperty',
+      of: 'org.chromium.apis.web.Browser',
       name: 'browser',
       documentation: `An Browser object used to calculate failure to ship
         metric with other verndors at this.date.`,
       required: true,
       final: true,
     },
     {
-      name: 'browserVersion',
-      class: 'String',
-      documentation: 'Version of this browser.',
-      required: true,
-      final: true,
-    },
-    {
-      name: 'date',
       class: 'Date',
-      documentation: 'The date at which this metric is computed.',
+      name: 'date',
+      documentation: 'The date for which this metric is computed.',
       required: true,
       final: true,
     },
     {
-      name: 'numFailureToShip',
       class: 'Int',
+      name: 'numFailureToShip',
       documentation: 'The number of failure-to-ship APIs.',
       required: true,
       final: true,
     },
     {
+      class: 'Array',
```

Comment:
Here and elsewhere: Please add `of: <package.path>.Browser` (by `<package.path>` I mean whatever is the actual package path to the `Browser` class).